docs/11-circleci/04-test.mdx

```diff
@@ -0,0 +1,336 @@
+---
+toc_max_heading_level: 4
+---
+
+# Test
+
+The Unity orb provides the
+[test job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-test) to facilitate the
+process of testing your Unity project. At the end of the run, the test results are parsed to JUnit
+and [stored](https://circleci.com/docs/collect-test-data) in CircleCI to enable
+[test insights](https://circleci.com/docs/insights-tests). You can access them from the CircleCI
+Dashboard.
```

Comment:
```suggestion
The Unity Orb provides the
[test job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-test) to facilitate the
process of testing your Unity project. 

At the end of the run, the test results are parsed to JUnit
and [stored](https://circleci.com/docs/collect-test-data) in CircleCI so that [test insights](https://circleci.com/docs/insights-tests) can be used. You can access test insights from the CircleCI
Dashboard.
```